website/content/docs/use-cases/hcp-hashistack-quickstart.mdx

```diff
@@ -0,0 +1,495 @@
+and set the environment variables for HCP and AWS, you are now ready to deploy your infrastructure!
+
+```shell
+$ terraform init
+Initializing the backend...
+Initializing provider plugins...
+...
+Terraform has been successfully initialized!
+...
+```
+
+Once Terraform has been initialized, you can verify that the resources will
+be created using the `plan` command.
+
+```shell
+$ terraform plan
+An execution plan has been generated and is shown below.
+Resource actions are indicated with the following symbols:
+  + create
+ <= read (data resources)
+Terraform will perform the following actions:
+...
+...
+Plan: 11 to add, 0 to change, 0 to destroy.
+```
+
+Finally, this is the magic moment I promised:
+
+```shell
+$ terraform apply
+...
+Do you want to perform these actions?
+  Terraform will perform the actions described above.
+  Only 'yes' will be accepted to approve.
+  Enter a value:
+```
+
+Remember to confirm the run by entering `yes`!
+
+## Set up HCP Waypoint Context
+
+HCP Waypoint is currently in Beta, so we don't have a convenient Terraform module to provision Waypoint automatically yet (as of February 2023).
+
+Head into your HCP UI and [follow this tutorial](https://developer.hashicorp.com/waypoint/tutorials/runners/hcp-runners-on-your-infra#set-up-the-environment)
+to set up your local environment.
+
+Navigate to Waypoint in the HCP dashboard:
+![HCP UI](img/use-cases/hcp-hashistack-quickstart/hcp.png)
+
+Then click on "Install New Runner", which will generate terminal inputs to set up the Waypoint context and install a Runner on your chosen platform.
+In this case, we choose Nomad with a host volume named "wp_runner".
+![Waypoint Install New Runner](img/use-cases/hcp-hashistack-quickstart/waypoint-runner-install.png)
+
+You will need to export the Nomad server address, which you can find on the AWS EC2 instance:
+![Nomad EC2 Instances](img/use-cases/hcp-hashistack-quickstart/nomad-ec2.png)
+
+```shell
+export NOMAD_ADDR=<nomad_private_addr>
+```
+
+![Waypoint Install New Runner CLI](img/use-cases/hcp-hashistack-quickstart/wp-runner-install.png)
+
+Export the nomad address to your local environment and install the runner after installing the HCP Waypoint context,
+and you're all set to use HCP Waypoint.
+
+The `waypoint.hcl` file defines application deployment configurations.
+For this use case, we have a simple "build", "deploy", and "release" flow:
+
+```hcl
+project = "2048-hashistack"
+
+app "nomad" {
+  build {
+    use "docker" {}
+    registry {
+      use "docker" {
+        ...
+      }
+    }
+  }
+
+  deploy {
+    use "nomad" {
+      datacenter = "hcp-2048"
+    }
+  }
+}
+```
+
+You can [read more about Waypoint HCL](https://developer.hashicorp.com/waypoint/docs/waypoint-hcl) fundamentals,
+or explore [advanced deployment workflows using custom pipelines](https://developer.hashicorp.com/waypoint/docs/pipelines).
+
+Now, run `waypoint init`:
```

Comment:
We should come back and update this to point to the WP Terraform Provider once Clint has migrated it!
https://github.com/hashicorp-dev-advocates/terraform-provider-waypoint